Source/WebCore/PAL/pal/PALSwift.h

```diff
@@ -28,7 +28,7 @@
 #include <cstdint>
 #include <wtf/Vector.h>
 
-using VectorUInt8 = WTF::Vector<uint8_t>;
+using VectorUInt8 = WTF::Vector<uint8_t, 0, CrashOnOverflow, 16, WTF::VectorBufferMalloc, WTF::IsCopyable::No>;
```

Comment:
Can't we just pass std::spans to Swift? Why do we need to pass vectors to these functions?

Reply:
If we'd pass a std::span, we wouldn't care if Swift copies it since it is cheap to copy

Reply:
Removed the nonCopyable and using spans instead.